addons/options/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
+  "types": "dist/public_api.d.ts",
```

Comment:
Why is this not `dist/index.d.ts`?

Reply:
Because this package has multiple files with exports - I then use public_api.d.ts as entry point and export everything from one file